### ISSUE TITLE
Is the COCO dataset only used for training segmentation models? Do tracking datasets require separate annotations?

### ISSUE THREAD

**Issue by DongLZY** [zhang-tao-whu/DVIS#25]

**Comment by DongLZY** [comment]
Hello Mr. Zhang, I would like to perform instance tracking for fruits. Is it possible to achieve this using DVIS? I have already created a dataset for image segmentation and can train instance segmentation models. Do I need to annotate the tracking training dataset additionally, or is it possible to generate it directly?

**Comment by zhang-tao-whu** [comment]
Yes, DVIS can handle this task. You can try training DVIS using only an image segmentation dataset, but its performance may not be optimal for complex occlusion scenarios. The best advice is to combine your fruit image segmentation dataset with an OVIS dataset for joint training.

**Comment by DongLZY** [comment]
Thank you for your answer. Can I use other segmener like yolact? I need to improve the processing speed of the model.

**Comment by zhang-tao-whu** [comment]
I think it's okay, but I haven't tried it.
